### PR TITLE
Add CMV

### LINKED ISSUE #17: Create CMV method
Method to call all conditions methods and create the Condition Met Vector

### PR DESCRIPTION
**Pull request by hampfh**
This creates a hard-coded list with the results from each of the conditions

### CODE REVIEW

File: decide/src/main/java/org/example/Decide.java [hunk 1]
```diff
@@ -12,8 +12,31 @@ public void decide() {
...
+                condition11(),
+                condition12(),
+                condition13(),
+                condition14()
+        };
+    }
+
     public boolean decideHelper() {
         // TODO do stuff here
+        // Create a list of booleans, one for each LIC
```
Comment (paulinegarelli) [inline decide/src/main/java/org/example/Decide.java]: Maybe this comment should be in the getConditionMetVector method

**Review by paulinegarelli — COMMENTED** [review]
Just the question of the comments, otherwise lgtm

File: decide/src/main/java/org/example/Decide.java (hunk 1 above)
Comment (hampfh) [reply decide/src/main/java/org/example/Decide.java]: That's a good point, I actually forgot that was there, should probably remove the "todo" too hehe

**Review by paulinegarelli — APPROVED** [review]
Good for me now

**Pull request merged**